docs/guides/nrfconnect_examples_software_update.md

```diff
@@ -0,0 +1,129 @@
+
+The following examples for the development kits from Nordic Semiconductor
+support over-the-air Device Firmware Upgrade:
+
+-   [CHIP nRF Connect Lock Example Application](../../examples/lock-app/nrfconnect/README.md)
+-   [CHIP nRF Connect Lighting Example Application](../../examples/lighting-app/nrfconnect/README.md)
+
+The
+[CHIP nRF Connect Pigweed Example Application](../../examples/pigweed-app/nrfconnect/README.md)
+does not support DFU.
+
+Currently, the Bluetooth LE is the only available transport for performing the
+DFU operation and it uses
+[Simple Management Protocol](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/device_mgmt/index.html#device-mgmt).
+The upgrade can be done either using a smartphone application or a PC command
+line tool.
+
+## Device Firmware Upgrade using smartphone
+
+To upgrade your device firmware over Bluetooth LE using smartphone, complete the
+following steps:
+
+1. Install one of the following applications on your smartphone:
+
+    - [nRF Connect for Mobile](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Connect-for-mobile)
+    - [nRF Toolbox](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Toolbox)
+
+2. Push **Button 1** on the device to enable the software update functionality.
+3. Push **Button 4** on the device to start the Bluetooth LE advertising.
+4. Follow the instructions about downloading the new image to a device on the
+   [FOTA upgrades](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_nrf52.html#fota-upgrades)
+   page in the nRF Connect documentation.
+
+## Device Firmware Upgrade using PC command line tool
+
+To upgrade your device firmware over Bluetooth LE, you can use the PC command
+line tool provided by the [mcumgr](https://github.com/zephyrproject-rtos/mcumgr)
+project.
+
+> **_WARNING:_** mcumgr tool using Bluetooth LE is available only for Linux and
+> macOS systems. On Windows, there is no support for Device Firmware Upgrade
+> over Bluetooth LE yet.
+
+Complete the following steps to perform DFU using mcumgr:
+
+1. Install the tool by following the
+   [mcumgr command line tool installation instructions](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/device_mgmt/index.html#command-line-tool).
+2. Push **Button 1** on the device to enable software update functionality.
+3. Push **Button 4** on the device to start the Bluetooth LE advertising.
+
+> **_NOTE:_** In all of the commands listed in the following steps, replace
+> `ble-hci-number` with the Bluetooth hci integer value (for example, `0`) and
+> `ble-device-name` with the CHIP device name advertised over Bluetooth LE (for
+> example, `ChipLock`).
+
+4.  Upload the firmware image to the device by running the following command in
+    your example directory:
+
+        $ sudo mcumgr --conntype ble --hci ble-hci-number --connstring peer_name='ble-device-name' image upload build/zephyr/app_update.bin
```

Comment:
nit: This is okay, but personally i prefer example commands which already have default values like:

`--hci 0 --connstring peer_name='ChipLock'`

..and only instructs the user to change them (as you did above in the note).